dist/recipe.nsi

```diff
@@ -77,14 +78,40 @@
 Section "Install"
  
   ;Add files
-  SetOutPath "$INSTDIR\generated"  
   SetOutPath "$INSTDIR"
```

Comment:
Yes, the Sidecar is dropping the generated configurations there. After the first start you can find some files there.